cmd/agent/common/common.go

```diff
@@ -68,3 +72,13 @@ func GetVersion(w http.ResponseWriter, r *http.Request) {
 	j, _ := json.Marshal(av)
 	w.Write(j)
 }
+
+// NewSettingsClient returns a configured runtime settings client.
+func NewSettingsClient() (settings.Client, error) {
+	ipcAddress, err := config.GetIPCAddress()
+	if err != nil {
+		return nil, err
+	}
+	hc := util.GetClient(false)
+	return settingshttp.NewClient(hc, fmt.Sprintf("https://%v:%v/agent/config", ipcAddress, config.Datadog.GetInt("cmd_port")), "datadog-agent"), nil
```

Comment:
nit: we generally refer to the core agent process as `agent` rather than `datadog-agent`

```suggestion
	return settingshttp.NewClient(hc, fmt.Sprintf("https://%v:%v/agent/config", ipcAddress, config.Datadog.GetInt("cmd_port")), "agent"), nil
```